input/pagecontent/changes.md

```diff
@@ -1,6 +1,17 @@
+###  Release 1.0.0
+- Expected Publication date: 2025-01
+- Expected Publication status: Normative
```

Comment:
This fix doesn't need a change log entry. Remove changes to changes.md from this PR.